README.md

```diff
@@ -219,6 +219,14 @@ To update a project, run the following command from its directory:
 ```bash
 bin/console upgradability:php:upgrade
 ```
+
+## Enable project change integration
+​
+* Turn on integrator trigger after package update step
+```bash
+export INTEGRATOR_ENABLED=true
```

Comment:
In this case, I think we can use --no-install if if INTEGRATOR_ENABLED=false. @DmytroKlymanSpryker

Reply:
By the technical spec: we don't need the dependency. We coud have a situation when we want to turn on dependency installation and turn off the integrator
Of course, it can be changed
for me better to have separate configurations for each feature